fractal-puppy/Engine
# FEATURE/ Loot Script V2

Added a new script to spawn prefabs of Items. It can be attached to an Enemy or Chest and once dead or opened it will drop the chosen Items. You only need to tell the name of the prefabs you want to spawn to the script.

Items can be spawned following a circle or semicircle around the spawner and have a delay timer to spawn it after chest open animation or enemy death animation has ended.

A script to scale GO over time also has been added. This can be applied to the loot to animate it.  


**To test:**

1. Open the App and load ItemsTest scene.
2. Hit play and advance until you reach a chest, touch it and see if it drops loot in front of it.
3. Kill the first enemy and see if it drops another item.
4. Kill the second enemy and see if it drops more items around with the scale animation.
5. Check that the items picked work.
